tools/check_csr.sh

```diff
@@ -41,7 +41,7 @@ match_csr() {
 
 CSR="$1"
 DN="$(openssl req -in "$1" -subject -noout)"
-HOST="$(echo $DN | sed 's/subject=[\/]*CN=//')"
+HOST="$(echo $DN | sed 's/subject=[\/]*CN\s*=\s*//')"
```

Comment:
Doesn't work on my mac. I think you need this:

```HOST="$(echo $DN | sed 's/subject=[\/]*CN[[:space:]]*=[[:space:]]*//')"```

Because \s [doesn't work on mac.](https://www.ojisanseiuchi.com/2020/08/13/sed-matching-whitespace-on-macos/)

Reply:
A little surprised it still works with `[[:space:]]` on my Ubuntu machine, updated the PR.